packages/ui-kit/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  extends: ['@commercetools-frontend/eslint-config-mc-app'],
+  rules: {
+    'prettier/prettier': [
+      'error',
+      {
+        parser: 'typescript',
+      },
+    ],
+  },
```

Comment:
something is wrong here I bet.  This is one of the ways I tried to get prettier to not crash on certain ts syntax.

Reply:
What was the error you got?